packages/ra-core/src/form/useSuggestions.ts

```diff
@@ -265,37 +246,6 @@ export const getSuggestionsFactory = ({
     );
 };
 
-/**
- * @example
- *
- * removeAlreadySelectedSuggestions(
- *  [{ id: 1, name: 'foo'}, { id: 2, name: 'bar' }],
- *  [{ id: 1, name: 'foo'}]
- * );
- *
- * // Will return [{ id: 2, name: 'bar' }]
- *
- * @param suggestions List of suggestions
- * @param selectedItems List of selection
- * @param getChoiceValue Converter function from suggestion to value
- */
-const removeAlreadySelectedSuggestions = (
```

Comment:
I don't understand why we can't keep this feature, it doesn't seem dependent on Mui Autocomplete

Reply:
Because mui needs to have all choices present, even those that are selected